### PR TITLE
Added watch command

### PR DESCRIPTION
**Pull request by tmcconechy**
Fixed Csp Issue
Fixed issue on accordion sample page

> Explain the **details** for making this change. What existing problem does the pull request solve?

Add watch to npm command list

Fixed sample page:
http://localhost:4000/components/accordion/example-ajax.html

> **Related issue (required)**:

https://jira.infor.com/browse/SOHO-8046

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
     "quickstart": "npx grunt demo && node app/server.js",
     "build": "npx grunt",
     "stop": "./scripts/stop.sh",
+    "watch": "npx grunt watch",
```
Comment (d-g-h) [inline package.json]: Drop for `npm run build -- watch` in the docs?

Comment (tmcconechy) [reply package.json]: At first i thought that was the correct command too. But im trying to put `grunt watch` in the script commands the `npm run build -- watch` seems to just build and stop and not actually watch.

Comment (d-g-h) [reply package.json]: screen shot 2018-05-29 at 3 48 17 pm

**Comment by tmcconechy** [comment]
(thumbs) i didnt have a space before the --

**Pull request merged**